README.md

```diff
@@ -83,6 +83,13 @@ Run tests:
 APP_ENV=test ROUTER_URL=http://localhost:4899 bundle exec rake test
 ```
 
+If you want to get information about how long last each test:
+```
+TIME=true HTML=true APP_ENV=test ROUTER_URL=http://localhost:4899 bundle exec rake test
```

Comment:
We currently avoid the router in every tests ?

Reply:
Yes, I do not know why this is like that. Did we use to call router within tests ?

Reply:
No, it makes the tests sensitive to the technology/version/config/data of the router

Reply:
So what was this ever used for ?

Reply:
In any case I should have removed this !

Reply:
It was used before we manage to avoid the router calls in all the tests.